sanghyeok/LV_1/수박수박수박수박수박수.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+    let result ="";
+    for(let i = 0; i < n; i++) {
+        result += i % 2 === 0 ? "수" : "박"; 
+    }
+        return result;
+}
```

Comment:
```suggestion
function solution(n) {
    let result ="";
    for(let i = 0; i < n; i++) {
        result += i % 2 === 0 ? "수" : "박"; 
    }
        return result;
}
```
오!! 저는 if - else 문으로 작성해주었는데 상혁님은 삼항연산자를 사용하셔서 더 간단하게 작성하셨네요!! 🫢
저도 앞으로는 간단한게 짤 수 있도록 생각해봐야겠어요!! 🤔
코드 잘 보고 갑니다!! 👍